tests/parens/jsfmt.spec.mjs

```diff
@@ -1 +1,2 @@
 run_spec(import.meta, ["php"]);
+run_spec(import.meta, ["php"], { phpVersion: "8.4" });
```

Comment:
Do we really need this? The only differences would be related to `new`, which are covered in the `new` tests, right?

Reply:
Alright! Thanks :)